ExampleGallery/UAP/CameraEffectExample.xaml.cs

```diff
@@ -119,6 +126,17 @@ private void EffectCombo_SelectionChanged(object sender, SelectionChangedEventAr
             var task = SetEffect(effect);
         }
 
+        private void VariableEffectValueSlider_OnValueChanged(object sender, RangeBaseValueChangedEventArgs e)
+        {
+            if (sender == null || e == null || effectPropertySet == null) return;
```

Comment:
why do you need to check for sender or e being null? I'm pretty sure that can never happen.